### ISSUE TITLE
ledger nano support doesn't work with legacy addresses

### ISSUE THREAD

**Issue by twocell** [Synthetixio/synthetix-exchange#90]
Whatever version of web3.js you use in the exchange app does not properly work with ledger legacy addresses.  Oddly, the mintr app does work so they must be different code.  Your can reproduce the problem by trying to connect to a Ledger that is using a legacy ethereum address.  The only work around i could come up with seems to be to bind the legacy address to metamask, and then in the exchange app, connect with by pressing the metamask button instead of the ledger button.  I believe ledger defaults to legacy out of the box, so alot of people will probably run into this.  More background on legacy addresses here:

https://github.com/MyCryptoHQ/MyCrypto/issues/2070

Long story short - you need to fix the exchange app so that you can pick the ledger derivation path (legacy or bip44) like you can in the mintr app.
